js/renovation/button.tsx

```diff
@@ -1,7 +1,7 @@
-import { Component, Prop, React } from "../component_declaration/common";
+import { Component, Prop, React } from '../component_declaration/common';
```

Comment:
This  module can be moved into renovation folder

Reply:
Moved